function-app-container/entrypoint.sh

```diff
@@ -49,5 +49,20 @@ az webapp config container set -n "${WEB_APP_NAME}" -g "${RESOURCE_GROUP_NAME}"
 
 echo "Configured image details to Azure App Service"
 
+APP_KIND=`az resource show -n "${WEB_APP_NAME}" -g "${RESOURCE_GROUP_NAME}" --resource-type "Microsoft.Web/Sites" --query 'kind'` 
+echo "Web App type : ${APP_KIND}"
+
```

Comment:
You need not query for appKind here. Container will always be linux type. Just ensure the app setting is present for every function container deployment.